src/Fmt_ast.ml

```diff
@@ -1759,11 +1771,16 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
                , fmt_after_cmts
                , (fmt_op, [(Nolabel, arg)]) ))) )
   | Pexp_construct (({txt= Lident "::"; loc= _} as lid), Some arg) ->
+      let opn, cls =
+        let ( ! ) = Caml.format_of_string in
+        match c.conf.indicate_multiline_delimiters with
+        | `No -> (!"(", !")")
+        | `Space -> (!"( ", !" )") (* Why is a space used instead a hint? *)
+        | `Closing_on_separate_line -> (!"(", !"@;<1000 -2>)")
```

Comment:
This is around the `( :: )` constructor so this should be like `No`. Assuming this is not correct (but should never happen):
```ocaml
a (::
) b
```

Reply:
Agreed, fixed.